src/Microsoft.ML.Transforms/Text/TextFeaturizingEstimator.cs

```diff
@@ -31,10 +31,36 @@ namespace Microsoft.ML.Transforms.Text
     /// </summary>
     public interface IStopWordsRemoverOptions { }
 
-    // A transform that turns a collection of text documents into numerical feature vectors. The feature vectors are counts
-    // of (word or character) ngrams in a given text. It offers ngram hashing (finding the ngram token string name to feature
-    // integer index mapping through hashing) as an option.
-    /// <include file='doc.xml' path='doc/members/member[@name="TextFeaturizingEstimator "]/*' />
+    /// <summary>
+    ///  A transform that turns a collection of text documents into numerical feature vectors.
+    ///  The feature vectors are normalized counts of word and/or character ngrams (based on the options supplied) in a given tokenized text.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes. |
+    /// | Input column data type | [text](xref:System.ReadOnlyMemory<char[]>) |
```

Comment:
Does this work?

Reply:
I thought it was: xref:System.ReadOnlyMemory{System.Char}
As per Shahab's message in teams

---
In reply to: [277057901](https://github.com/dotnet/machinelearning/pull/3438#discussion_r277057901) [](ancestors = 277057901)

Reply:
thanks for the catch!

Reply:
actually i think we're doing TextDataViewType

---
In reply to: [277110296](https://github.com/dotnet/machinelearning/pull/3438#discussion_r277110296) [](ancestors = 277110296)

Reply:
I am putting the following in there:
Scream now, or ... :)


 | Input column data type | [text](xref:Microsoft.ML.Data.TextDataViewType) |

---
In reply to: [277111141](https://github.com/dotnet/machinelearning/pull/3438#discussion_r277111141) [](ancestors = 277111141,277110296)

Reply:
Checked with Shahab. The above  is correct. 

---
In reply to: [277111655](https://github.com/dotnet/machinelearning/pull/3438#discussion_r277111655) [](ancestors = 277111655,277111141,277110296)